src/benchmarks/gc/src/exec/env/make_memory_load.c

```diff
@@ -105,7 +106,9 @@ typedef struct Args
     bool no_readjust;
 } Args;
 
-const char* USAGE = "Usage: make_memory_load.exe -percent 50 [-neverRelease]\n";
+// TODO: Add Verbose/Debug option to show all the stats while running without
```

Comment:
If you think it is worth to follow up on this, there should be a bug/workitem instead of TODO. 
If you change your mind later, the bug could be closed without touching this file.

Reply:
I had not thought about it, but I agree it's the better option. I've removed this comment and open a new issue to track this task: https://github.com/dotnet/performance/issues/1225